website/docs/guides/setting-up-dex.md

```diff
@@ -0,0 +1,281 @@
+          - name: weaveworks
+            teams:
+            - team-a
+            - team-b
+            - QA
+          - name: ww-test-org
+      staticPasswords:
+      - email: "admin@example.tld"
+        hash: "$2a$10$2b2cU8CPhOTaGrs1HRQuAueS7JTT5ZHsHSzYiFPm1leZck7Mc8T4W"
+        username: "admin"
+        userID: "08a8684b-db88-4b73-90a9-3cd1661f5466"
+    ingress:
+      enabled: true
+      className: nginx
+      annotations:
+        cert-manager.io/cluster-issuer: letsencrypt-prod
+      hosts:
+        - host: dex.dev.example.tld
+          paths:
+          - path: /
+            pathType: ImplementationSpecific
+      tls:
+        - hosts:
+          - dex.dev.example.tld
+          secretName: dex-dev-example-tld
+```
+
+:::note SSL certificate without cert manager
+If we don't want to use cert manager, we can remove the related annotation and
+use our predefined secret in the `tls` section.
+:::
+
+An important part of the configuration is the `orgs` field on the GitHub
+connector.
+
+```yaml
+orgs:
+- name: weaveworks
+  teams:
+  - team-a
+  - team-b
+  - QA
+```
+
+Here we can define groups under a GitHub organisation. In this example the
+GitHub organisation is `weaveworks` and all members of the `team-a`,
+`team-b`, and `QA` teams can authenticate. Group membership will be added to
+the user.
+
+Based on these groups, we can bind roles to groups:
+
+```yaml
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: wego-test-user-read-resources
+  namespace: flux-system
+subjects:
+  - kind: Group
+    name: weaveworks:QA
+    namespace: flux-system
+roleRef:
+  kind: Role
+  name: wego-admin-role
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: wego-admin-role
+  namespace: flux-system
+rules:
+  - apiGroups: [""]
+    resources: ["secrets", "pods" ]
+    verbs: [ "get", "list" ]
+  - apiGroups: ["apps"]
+    resources: [ "deployments", "replicasets"]
+    verbs: [ "get", "list" ]
+  - apiGroups: ["kustomize.toolkit.fluxcd.io"]
+    resources: [ "kustomizations" ]
+    verbs: [ "get", "list", "patch" ]
+  - apiGroups: ["helm.toolkit.fluxcd.io"]
+    resources: [ "helmreleases" ]
+    verbs: [ "get", "list", "patch" ]
+  - apiGroups: ["source.toolkit.fluxcd.io"]
+    resources: [ "buckets", "helmcharts", "gitrepositories", "helmrepositories" ]
+    verbs: [ "get", "list", "patch" ]
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["get", "watch", "list"]
+```
+
+The same way we can bind cluster roles to a group:
+
+```yaml
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: weaveworks:team-a
+subjects:
+- kind: Group
+  name: weaveworks:team-a
+  apiGroup: rbac.authorization.k8s.io
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
+  apiGroup: rbac.authorization.k8s.io
+```
+
+### Set up static user
+
+For static user, add `staticPasswords` to the `config`:
```

Comment:
this is already in the config. should we mention the inverse instead?

Reply:
It's not in there anymore.